### DISCUSSION TITLE
Complex filtering

### DISCUSSION THREAD

**Discussion by mharis111** [mirumee/ariadne#500] [General]
Does Ariadne support aggregate functions? such as finding average, max and min values and, also use of <,> operators in some way. Looked into documentation but didn't find anything related to it. Sorry, If I skipped anything.

**Comment by rafalp** [comment]
Hey @mharis111!

GraphQL servers by design don't provide any aggregate functions (or any data functions). They only let you to define your own types and read/mutate them through `query` and `mutation` operations.

You may be mistaking Ariadne for database solutions exposing GraphQL APIs like Hasura o Prisma.
